.github/workflows/release_pull_request.yml

```diff
@@ -67,4 +67,16 @@ jobs:
         run: |
           echo "::set-output name=dev::${{env.DEV_PR}}"
           echo "::set-output name=release::${{env.RELEASE_HASH}}"
-
+          
+      - name: Notify Slack
+        if: ${{ failure() }}
+        env:
+          SLACK_API_TOKEN: ${{ secrets.SLACK_API_TOKEN }}
+          channel_name: va-mobile-build-alerts
+          message: 'The `/approve` command for release: ${{inputs.version}} has failed. See: <${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}|workflow run> for results.'
+        run: |
+            curl -X POST \
+                 -H "Authorization: Bearer $SLACK_API_TOKEN" \
+                 -H "Content-Type: application/json" \
+                 -d "{\"channel\":\"$channel_name\",\"text\":\"$message\"}" \
+                 https://slack.com/api/chat.postMessage
```

Comment:
We have the [start_slack_thread](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/.github/workflows/start_slack_thread.yml) reusable workflow that can be used for this

Reply:
done. Please check it out!